lib/buckets.ps1

```diff
@@ -39,12 +39,17 @@ function apps_in_bucket($dir) {
     return Get-ChildItem $dir | Where-Object { $_.Name.endswith('.json') } | ForEach-Object { $_.Name -replace '.json$', '' }
 }
 
+function Get-LocalBucket {
```

Comment:
The plural of `Get-LocalBuckets` would be better since it was `buckets`

Reply:
https://github.com/PowerShell/PSScriptAnalyzer/blob/development/RuleDocumentation/UseSingularNouns.md